code/go/0chain.net/miner/protocol_block.go

```diff
@@ -162,6 +162,8 @@ func (mc *Chain) validateTransaction(b *block.Block, bState util.MerklePatriciaT
 	}
 
 	if txn.Nonce-state.Nonce < 1 {
+		logging.Logger.Debug("past txn", zap.String("txn", txn.Hash),
```

Comment:
we probably need to check debugTxn for this log, since there can be a lot of such logs under the load